Dockerfile

```diff
@@ -537,24 +535,23 @@ RUN set -x; \
 		zypper -n install bison>3.4 && \
 		# Allow fluent-bit to find systemd
 		ln -fs /usr/lib/systemd /lib/systemd && \
-		# Download and install CMake manually (zypper only gives us CMake 3.5, but we need >= 3.12)
-		wget https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-x86_64.sh -O cmake.sh && \
-		(echo '4d98de8d605da676e71a889dd94f80c76abb377fade2f21e3510e62ece1e1ada  cmake.sh' | sha256sum --check) && \
-		bash cmake.sh --skip-license --prefix=/usr/local
-	
+		wget https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-$(uname -m).sh -O cmake.sh && \
+	((echo '4d98de8d605da676e71a889dd94f80c76abb377fade2f21e3510e62ece1e1ada  cmake.sh' | sha256sum --check) || \
+	(echo '73a35cab2174a3eb8f35083d55c80871185dc3808f3dae3558cd5fbdb29a4614  cmake.sh' | sha256sum --check)) && \
+	bash cmake.sh --skip-license --prefix=/usr/local
 		ADD https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_linux_hotspot_11.0.13_8.tar.gz /tmp/OpenJDK11U-jdk_x64_linux_hotspot_11.0.13_8.tar.gz
 		RUN set -xe; \
-			mkdir -p /usr/local/java-11-openjdk && \
-			tar -xf /tmp/OpenJDK11U-jdk_x64_linux_hotspot_11.0.13_8.tar.gz -C /usr/local/java-11-openjdk --strip-components=1
+		mkdir -p /usr/local/java-11-openjdk && \
```

Comment:
Should the indentation here be the previous?

Reply:
I'm not too concerned with the indentation in the Dockerfile since it's auto-generated; my preference is consistent indentation in `compile.go` since that file is the hand-written one. But I'm also fine to revert this to reduce the size of the diff.